HEROKU.md

```diff
@@ -35,16 +35,29 @@ heroku run bash
 ```
 Furthermore, you can access the Heroku Postgres database by running the following within that environment to execute SQL:
 ```
-psql <the DATABASE_URL from your config vars>
+psql <the DATABASE_URL located in your config vars under the Settings tab>
 ```
 Next we need to set up the database and insert our institution in to the appropriate table.
-1. Proceed with [setting up the database](https://github.com/ucfopen/UDOIT/blob/issue/570-heroku/INSTALL.md#database-setup).
-  Important Note: Before setting up the database and if using Postgres, ensure that the table name in 'UDOIT/src/Entity/User.php' at line 14 is set to 'users'.
-  This change must be contained in a commit and be pushed to the Heroku remote.
-2. Now is a good time to set up developer keys according to the instructions in INSTALL_\<LMS\>.md.
-3. Insert your institution in to the institution table as described in INSTALL_\<LMS\>.md.
+1. Before setting up the database and if using Postgres, ensure that the table name in 'UDOIT/src/Entity/User.php' at line 14 is set to 'users'.
+  This change must be contained in a commit and be pushed to the Heroku remote before continuing.
+2. If using Heroku Postgres, generate a migration by using:
+```
+php bin/console doctrine:migrations:diff
```

Comment:
Are these commands supposed to be run on the Heroku machine?  If so, add a step for running `heroku ps:exec`

Reply:
I moved the line for using `heroku run bash` into a step under the 'Database Migration and Setup' section. I decided not to go with `heroku ps:exec` based on this [post about using bash for database migrations](https://stackoverflow.com/questions/54386370/heroku-what-is-the-difference-between-heroku-psexec-and-heroku-run-bash).